### PR TITLE
[cmd/opampsupervisor] docs: improvements in the supervisor startup guide

### PR DESCRIPTION
**Pull request by kasia-kujawa**
**Description:** improvements in the supervisor startup guide
**Testing:** manual test of all instruction steps

### CODE REVIEW

**Review by dmolenda-sumo — APPROVED** [review]

File: cmd/opampsupervisor/README.md [hunk 1]
```diff
@@ -9,15 +9,25 @@ The design is still undergoing changes, and as such this implementation may chan
 The supervisor is currently undergoing heavy development and is not ready for any serious use. However, if you would like to test it, you can follow the steps below:
 
 1. Download the [opamp-go](https://github.com/open-telemetry/opamp-go) repository, and run the OpAMP example server in the `internal/examples/server` directory.
+
+   ```shell
+   git clone git@github.com:open-telemetry/opamp-go.git
+   cd internal/examples/server
```
Comment (astencel-sumo) [inline cmd/opampsupervisor/README.md:15]: If you just cloned it, you need to `cd` into the cloned directory too, right? :slightly_smiling_face: 
```suggestion
   cd opamp-go/internal/examples/server
```

File: cmd/opampsupervisor/README.md [hunk 2]
```diff
@@ -9,15 +9,25 @@ The design is still undergoing changes, and as such this implementation may chan
...
+   Visit [localhost:4321](http://localhost:4321) to verify that the server is running.
+
 2. From the Collector contrib repository root, build the Collector:
 
    ```shell
    make otelcontribcol
    ```
 
-3. Run the supervisor, substituting `<OS>` for your platform:
+3. Run the supervisor in the `cmd/opampsupervisor` directory of Collector contrib repository, substituting `<OS>` for your platform:
```
Comment (astencel-sumo) [inline cmd/opampsupervisor/README.md:27]:
```suggestion
3. Run the supervisor in the `cmd/opampsupervisor` directory of Collector contrib repository, substituting `<OS>` for your operating system (`darwin` for MacOS, `linux` or `windows`):
```

File: cmd/opampsupervisor/README.md (hunk 1 above)
Comment (kkujawa-sumo) [reply cmd/opampsupervisor/README.md]: changed

**Review by andrzej-stencel — APPROVED** [review]

**Review by songy23 — APPROVED** [review]

**Review by evan-bradley — APPROVED** [review]
Thanks @kkujawa-sumo!

**Pull request merged**